src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const headers = table.querySelectorAll('th');
+
+headers.forEach((header, index) => {
+  header.addEventListener('click', () => {
+    sortTable(index);
+  });
+});
+
+function sortTable(columnIndex) {
+  const tbody = table.querySelector('tbody');
+  const rows = Array.from(tbody.querySelectorAll('tr'));
+
+  rows.sort((a, b) => {
+    const cellA = a.querySelector(`td:nth-child(${columnIndex + 1})`);
+    const cellB = b.querySelector(`td:nth-child(${columnIndex + 1})`);
+
+    const valueA = cellA.textContent.trim();
+    const valueB = cellB.textContent.trim();
+
+    return valueA.localeCompare(valueB, undefined, { numeric: true });
+  });
+
+  // Очищаем tbody от существующих строк
```

Comment:
Remove comments. Try it write obvious code that explain all we need without comments)

Reply:
I have some issues with removing comments in code, I don't know why, but github doesn't add my new commits. Changed the Demo Link